.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "SpongeAPI"]
 	path = SpongeAPI
-	url = https://github.com/SpongePowered/SpongeAPI.git
+	url = https://github.com/kashike/SpongeAPI.git
```

Comment:
@windy1 This is how all non-core dev PRs have to be to get travis-ci to not fail the build, as stated by @ryantheleach.

It's split into two commits.